Eat_and_Greet/tsconfig.json

```diff
@@ -1,4 +1,3 @@
 {
-  "include": ["./assets/images/Logo.png"],
```

Comment:
Thank you for deleting this, I have created a better config file for us to use later.